packages/manager/modules/logs-to-customer/tsconfig.json

```
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "lib": ["dom", "es2020"],
+    "noEmit": true,
+    "target": "es2020",
+    "types": ["vite/client", "node"],
+    "module": "ES2020",
+    "moduleResolution": "node",
+    "removeComments": true,
+    "esModuleInterop": true,
+    "isolatedModules": true,
+    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
+    "noImplicitAny": true,
+    "declaration": true,
+    "resolveJsonModule": true,
+    "allowJs": true,
+    "jsx": "react"
```

To add @/... in all your imports

```suggestion
    "jsx": "react",
    "paths": {
      "@/*": ["./src/*"]
    }
```